install/kubernetes/helm/istio-remote/templates/clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+  kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: istio-reader
+  rules:
+    - apiGroups: ['']
+      resources: ['nodes', 'pods', 'services', 'endpoints']
```

Comment:
Do we need to add permission for gateway, destinationrules, virtualservices?

Reply:
No because those resources will only exist in the primary control plane cluster.

Reply:
@gyliu513  - I confirm  @Rigdon's answer

Reply:
yes, we have only one control panel.